src/main/kotlin/io/github/yuutoproject/yuutobot/Listener.kt

```diff
@@ -51,6 +51,14 @@ class Listener : ListenerAdapter() {
             return
         }
 
+        val botcmdsChannel = Yuuto.config["BOTCMDS_${event.guild.idLong}"]
+
+        // if the botcms channel is set for the server and the channel is the channel stored
+        // commands can be eecuted, if not this will return
+        if (botcmdsChannel != null && botcmdsChannel != event.channel.id) {
+            return
+        }
+
```

Comment:
```suggestion
        val botcmdsChannel = Yuuto.config["BOTCMDS_${event.guild.idLong}"]

        // If the BOTCMDS channel is set for the server and the channel is the channel stored
        // Commands can be executed, if not this will return
        if (botcmdsChannel != null && botcmdsChannel != event.channel.id) return
```